CHANGELOG.md

```diff
@@ -1,3 +1,122 @@
+### citus v12.0.0 (July 11, 2023) ###
+
+PR #6866 : Add support for schema-based-sharding via a GUC
+TODO: PLEASE SHORTEN THE NEXT LINE MANUALLY, IT SHOULD BE NO LONGER THAN 78 CHARS
+* Adds citus.enable_schema_based_sharding GUC that allows sharding the database based on schemas when enabled.
+
+* Adds `citus_schema_distribute` and `citus_schema_undistribute` UDFs for
+  conversion between regular and distributed schemas (#6933)
+
+* Adds support for creating distributed tables without shard key (#6867)
+
+* Enable logical planner for single-shard tables (#6950)
+
+* Add citus_schemas view (#6979)
+
+* Enabling citus_stat_tenants to support schema-based tenants. (#6936)
+
+* Support CREATE TABLE .. AS SELECT .. commands for tenant tables (#6998)
+
+* Disallow single shard table creation if replication factor > 1 (#7006)
+
+PR #7027 : Improve error/hint messages related to schema-based sharding
+
+PR #6971 : Fix citus_table_type column in citus_tables and citus_shards views for single shard tables
+
+PR #6965 : Disable some udfs for tenant tables
+
+PR #6963 : Single Shard Misc UDFs 2
+
+PR #6924 : Add Support for Single Shard Tables in update_distributed_table_colocation
+
+PR #6916 : Citus UDFs support for single shard tables
+
+PR #6908 : Prevent downgrades when there is a single-shard table in the cluster
+
+PR #6902 : Disable master_create_empty_shard udf for single shard tables
```

Comment:
@onurctirtir It would be better if you take a look at all those items which are related to schema sharding. Do we miss some PRs like https://github.com/citusdata/citus/pull/6823, https://github.com/citusdata/citus/pull/6752, and DDL disabling for single shard?

Reply:
internally discusse and updated